### PR TITLE
use towncrier to generate newsfile

### PR DESCRIPTION
**Pull request by davidhyman**

### CODE REVIEW

**Comment by davidhyman** [comment]
I think your comment went missing ...

File: NEWS.rst [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Mbed Cloud SDK for Python
+=========================
+The latest changes to the SDK - please see `mbed-cloud-sdk <https://pypi.org/project/mbed-cloud-sdk/#history>`__ for the
+versions actually released to PyPI.
```
Comment (madchutney) [inline NEWS.rst]: This sentence didn't scan when I read it, although I know what it means.

File: scripts/generate_news.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+def main():
+    """Writes out newsfile if significant version bump"""
+    PROJECT_ROOT = os.path.dirname(os.path.dirname(__file__))
+    metafile = os.path.join(PROJECT_ROOT, 'docs', 'news', 'last_built.meta')
+    last_known = '0'
+    if os.path.isfile(metafile):
+        with open(metafile) as fh:
+            last_known = fh.read()
+
+    import mbed_cloud
```
Comment (madchutney) [inline scripts/generate_news.py]: Why the import in the middle of the code?

File: scripts/notify.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import os
```
Comment (madchutney) [inline scripts/notify.py]: Missing doc string

**Pull request merged**